sklearn/linear_model/_logistic.py

```diff
@@ -828,6 +834,7 @@ class LogisticRegression(LinearClassifierMixin, SparseCoefMixin, BaseEstimator):
     'saga' solver.
 
     Read more in the :ref:`User Guide <logistic_regression>`.
+    https://scikit-learn.org/regressionstable/modules/linear_model.html#logistic-
```

Comment:
This is not a valid link. It is also not in the correct place. Instead, put

```
    For an example of how to use logistic regression, see:
    :ref:`sphx_glr_auto_examples_linear_model_plot_logistic.py`.
```

above the `Read more in the user guide` line, with one empty line in between (again, remember not to leave any whitespace on that empty line).